cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -91,5 +94,19 @@ func updateAllInstanceAccessConfig(workflow *daisy.Workflow, accessConfigProvide
 			}
 		}
 	}
+}
+
+// RemovePrivacyLogInfo removes privacy log information.
+func RemovePrivacyLogInfo(message string) string {
+	// Since translation scripts are vary and is hard to predict the output, we have to hide the
+	// details and only remain "TranslateFailed"
+	if strings.Contains(message, "TranslateFailed") {
```

Comment:
No unfortunately, since it's just a string. We have to do pattern match. Similarly "StatusMatch", "SuccessMatch" and "FailrueMatch" have the same problem.

Reply:
At least make it configurable and extract failure prefixes to constants